### PR TITLE
fix for middle clicking links

### PR DESCRIPTION
**Pull request by cokert**
This is a fix for #877.  There are some questions on the original thread that I asked, but here's what I've done so far for feedback.

### CODE REVIEW

**Comment by mondalaci** [comment]
Thank you very much for your contribution! CI failed because the linter complained. Would you please fix these stylistic issues?

File: packages/uhk-common/src/util/constants.ts [hunk 1]
```diff
@@ -2,5 +2,6 @@ export namespace Constants {
     export const AGENT_GITHUB_URL = 'https://github.com/UltimateHackingKeyboard/agent';
     export const AGENT_CONTRIBUTORS_GITHUB_PAGE_URL = 'https://github.com/UltimateHackingKeyboard/agent/graphs/contributors';
     export const AGENT_CONTRIBUTORS_GITHUB_API_URL = 'https://api.github.com/repos/UltimateHackingKeyboard/agent/contributors';
+    export const AGENT_HELP_ACCENT_CHARS_URL = 'https://ultimatehackingkeyboard.com/blog/2018/06/23/how-can-i-type-accented-characters-with-my-uhk'
```
Comment (ert78gb) [inline packages/uhk-common/src/util/constants.ts]: This constant is never used. Why did you add it?

Comment (cokert) [reply packages/uhk-common/src/util/constants.ts]: It is in the help page.  It was originally embedded directly in the HTML of the component.  I didn't change it in this original commit/PR, but it's should be used now.  It's discussed in comments of the original issue thread.

File: packages/uhk-common/src/util/constants.ts [hunk 2]
```diff
@@ -2,6 +2,6 @@ export namespace Constants {
     export const AGENT_GITHUB_URL = 'https://github.com/UltimateHackingKeyboard/agent';
     export const AGENT_CONTRIBUTORS_GITHUB_PAGE_URL = 'https://github.com/UltimateHackingKeyboard/agent/graphs/contributors';
     export const AGENT_CONTRIBUTORS_GITHUB_API_URL = 'https://api.github.com/repos/UltimateHackingKeyboard/agent/contributors';
-    export const AGENT_HELP_ACCENT_CHARS_URL = 'https://ultimatehackingkeyboard.com/blog/2018/06/23/how-can-i-type-accented-characters-with-my-uhk'
+    export const AGENT_HELP_ACCENT_CHARS_URL = 'https://uhk.io/blog/2018/06/23/how-can-i-type-accented-characters-with-my-uhk';
```
Comment (mondalaci) [inline packages/uhk-common/src/util/constants.ts]: I'd rather prefer ultimatehackingkeyboard.com. We only use our uhk.io in onboarding URLs. Using the full URL also makes opening such links faster, as no redirection is involved.

**Comment by cokert** [comment]
In this updated commit I've changed all the links to use the externalUrl directive.  I also found a link in the Firmware page that isn't visible by default.

File: packages/uhk-common/src/util/constants.ts (hunk 2 above)
Comment (cokert) [reply packages/uhk-common/src/util/constants.ts]: That puts the URL over the linter's limit.  I could break this into separate lines?  

But that brings up the question again of do you guys WANT all URLs in the constants file?  It seems kind of weird to have FAQ links embedded in the constants file.  For those, I'd think it better to have them just embedded in the HTML like it was originally.  Especially if you want to add more links in the help page over time.

Comment (mondalaci) [reply packages/uhk-common/src/util/constants.ts]: Sure, you can break overly large strings into separate lines.

I agree that it's simpler to embed such URLs in the HTML. Please do.

**Comment by cokert** [comment]
Ok, the FAQ link is back to being embedded in the HTML.   :)

**Comment by ert78gb** [comment]
@cokert Thx for the PR. Could you create an empty commit to start the travis builds.

**Comment by cokert** [comment]
I signed the CLA last night, but it's still reporting as pending.  Is there something you guys need to do on your side to move this forward?

**Comment by mondalaci** [comment]
@cokert I've just removed CLA assistant because of a [standing issue](https://github.com/UltimateHackingKeyboard/agent/issues/931#issuecomment-483848667). We'll implement CLA signing later, and it shouldn't be needed in the meantime. Can you commit now?

**Comment by cokert** [comment]
Ok, empty commit pushed, CI started.

**Comment by cokert** [comment]
Is there anything else I need to do on this from my end?

**Comment by mondalaci** [comment]
@cokert Just merged this PR. Sorry for the delay, and thank you very much for your contribution!
